src/library/scala/collection/SortedMap.scala

```diff
@@ -114,7 +114,7 @@ trait SortedMapOps[K, +V, +CC[X, Y] <: Map[X, Y] with SortedMapOps[X, Y, CC, _],
   override def keySet: SortedSet[K] = new KeySortedSet
 
   /** The implementation class of the set returned by `keySet` */
-  protected class KeySortedSet extends SortedSet[K] with GenKeySet with GenKeySortedSet {
+  protected class KeySortedSet extends AbstractSet[K] with SortedSet[K] with GenKeySet with GenKeySortedSet {
```

Comment:
What's the actual convention here? Why AbstractSet rather than AbstractSortedSet?

Reply:
There is no `AbstractSortedSet`, or I'd've used it.  Not 100% sure what the convention is, but I can introduce and use (throughout) `AbstractSortedSet` if preferred.

Reply:
you can't actually introduce it unless this is making it into 2.13.0

Reply:
in fact, checking the build output, even this change is not binary compatible

Reply:
Yup, it's forwards-incompatible, unless we're happy with, for instance, `scala.<:<` having a `scala.runtime.AbstractFunction1` parent in 2.13.1.  (`<:<` is sealed, so particularly unproblematic, but the others aren't).

I think `MissingTypesProblem` is a problem if it imposes abstract definitions downstream, such as having to implement a method.  There is no such imposition, so it should be fine, no?  (I'm unsure.)

Reply:
No it's not fine, due to type inference in lubbing. Such a change was erroneously thought she in 2.11.9 or 2.11.10, and was the reason that one of those two versions was blacklisted.

You can't do those changes in a minor Scala release.

In addition, I think it's really bad anyway to reference `runtime` in source code, especially in what amounts to public API. `runtime` is an implementation detail of the language and compiler. It should even be hidden from the rest of the standard library.